CoreFoundation/URL.subproj/CFURLSessionInterface.c

```diff
@@ -366,6 +374,7 @@ CFURLSessionOption const CFURLSessionOptionCOOKIE = { CURLOPT_COOKIE };
 CFURLSessionOption const CFURLSessionOptionHTTPHEADER = { CURLOPT_HTTPHEADER };
 CFURLSessionOption const CFURLSessionOptionHTTPPOST = { CURLOPT_HTTPPOST };
 CFURLSessionOption const CFURLSessionOptionSSLCERT = { CURLOPT_SSLCERT };
+CFURLSessionOption const CFURLSessionOptionSSLCERT_BLOB = { CURLOPT_SSLCERT_BLOB };
```

Comment:
Same comment as above, let's guard this option and `CURLOPT_SSLKEY_BLOB` below for libcurl versions that don't support it